README.md

```diff
@@ -4,6 +4,8 @@ If you are unfamiliar with PerfView, there are [PerfView video tutorials](http:/
 As well as [Vance Morrison's blog](http://blogs.msdn.com/b/vancem/archive/tags/perfview) which also gives overview and getting 
 started information. 
 
+[![Build status](https://ci.appveyor.com/api/projects/status/fxtu3xa874whk2w0?svg=true)](https://ci.appveyor.com/project/sharwell/perfview)
```

Comment:
> My experience is that with documentation you should err on the side of too much rather than too little 

I agree, and I don't mind adding this documentation. I will put something together with the intent of being the minimal amount of "stuff to read" (on the assumption that people have short attention spans) while still conveying the important information clearly.

> I mainly was interested in how trustworthy the output artifacts are (do we expect people to run the output from it, and if so, what risk are they taking?).

The build output is attached to each build. There is no guarantee whatsoever regarding the quality or security of these builds; any member of the public who is able to submit a pull request is able to get this system to create a build artifact.